elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,5 +1,7 @@
 @import "../../pfe-sass/pfe-sass";
 
+$LOCAL: cta;
```

Comment:
@starryeyez024 Should not be any visual changes with this update, just adjusting the code to be more efficient; this was part of getting the autoprefixer to work on all files (there were syntax errors in here I needed to fix for it to run).

Reply:
hmm, maybe there's a syntax error somewhere along the way, i'm seeing some style diffs
https://www.evernote.com/l/AA4EpESUToRJHZUUsn5ZMBbNrvPCn3PU0lk

Reply:
Yeah, I'm not sure you have the latest.

Reply:
I do, I'm on commit hash 736c343f1d73f1a9c730722b9d5751ddcf15944c from Feb 22 where Michael merged in master. 

I am seeing a lot of changes in the cta.scss file in this PR, including lots more CSS variables and some styles moved around. Previously there were more `!important` declarations which I think prevented the cp-theme styles from trouncing on the links in the light dom of the pfe-cta. 
![image](https://user-images.githubusercontent.com/456863/53364635-3ea6fd00-390d-11e9-9a2a-45df1173df2c.png)

Do we need these changes to make autoprefixer or ie11 work or could we break these out into a separate PR for now?

Reply:
I pulled out the updates.  We can debug the inconsistencies another time.